redis-formula/redis/init.sls

```diff
@@ -0,0 +1,58 @@
+{#-
+Salt state file for managing Redis
+Copyright (C) 2023 SUSE LLC <georg.pfuetzenreuter@suse.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- from 'redis/map.jinja' import config, dirs -%}
+
+include:
+  - .package
```

Comment:
package.sls consists of a single `pkg.installed` + the usual overhead.

Is this really worth the separate file, or would it make more sense to inline the `pkg.installed` into init.sls?

Reply:
I oriented myself at the multipath formula which has the same setup. I think it's mostly for convenience, being able to update the configuration without waiting for zypper to do its thing. Though either way is fine with me, which one do you prefer?

Reply:
I'd prefer to avoid the overhead of having a separate file for a single `pkg.installed` ;-)

(I'm also not sure if there's a performance difference between including a separate file and having pkg.installed inlined. I never tested, but I'd expect no difference.)

Reply:
```
$ time salt-call --local state.show_sls redis saltenv=devel

# with the separate file included
Executed in    1.33 secs    fish           external
   usr time    1.13 secs  375.00 micros    1.13 secs
   sys time    0.14 secs  244.00 micros    0.14 secs

# without the include, pkg.installed inside init.sls
Executed in    1.29 secs    fish           external
   usr time    1.13 secs    0.00 micros    1.13 secs
   sys time    0.09 secs  679.00 micros    0.09 secs
```

Fluctuates with 1-2 second differences.
I assume the separate Jinja render operation plus some related logic makes it slower.

Reply:
Consolidated via 5466eef66df9a237cce0eacd6f2e41653fe6f3ae.